Specs/NumberWithUnit/English/DimensionModel.json

```diff
@@ -1234,6 +1234,38 @@
       }
     ]
   },
+  {
+    "Input": "The weight is 150 microgram",
+    "Results": [
+      {
+        "Text": "150 microgram",
+        "Start": 14,
+        "End": 26,
+        "TypeName": "dimension",
+        "Resolution": {
+          "subtype": "Weight",
+          "unit": "Microgram",
+          "value": "150"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "The weight is 120 μg",
+    "Results": [
+      {
+        "Text": "120 μg",
+        "Start": 14,
+        "End": 19,
+        "TypeName": "dimension",
+        "Resolution": {
+          "subtype": "Weight",
+          "unit": "Microgram",
+          "value": "120"
+        }
+      }
+    ]
+  },
```

Comment:
Minor: When adding new cases, please always append at the end of the spec files. This makes merging easier.
Also, do not create a branch inside this repo. It's preferable to fork the repo and create branches for each PR there.